### PR TITLE
Future branch lesson 14 fix

### PR DESCRIPTION
**Pull request by YurySuvalau**

### CODE REVIEW

File: src/test/java/page/AccountListPage.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public AccountListPage(WebDriver driver) {
+        super(driver);
+    }
+
+    By NEW_BUTTON = By.cssSelector("[title=New]");
+    String NAME_ACCOUNT = account.getAccountName().split(" ")[0];
+    By TITLE_NAME = By.cssSelector("[title~=" + NAME_ACCOUNT + "]");
+
+    @Override
+    public AccountListPage waitForPageOpened() {
```
Comment (vchesnakoutest) [inline src/test/java/page/AccountListPage.java]: if you're not using 'chain of invocations', make this method return void (and other waiters as well)

File: src/test/java/page/AccountListPage.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        openPage(URL_LOGIN + URL_ACCOUNT_LIST);
+    }
+
+    public void clickNewButton() {
+        driver.findElement(NEW_BUTTON).click();
+    }
+
+    public boolean isAccountNameDisplayed() {
+        String label = driver.findElement(TITLE_NAME).getText();
+        if (label.contains(NAME_ACCOUNT)) {
```
Comment (vchesnakoutest) [inline src/test/java/page/AccountListPage.java]: just return label.contains(NAME_ACCOUNT)

File: src/test/java/page/HomePage.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+//        wait.until(ExpectedConditions.elementToBeClickable(ACCOUNT_BUTTON));
+//        JavascriptExecutor executor = (JavascriptExecutor) driver;
+//        executor.executeScript("arguments[0].click();", driver.findElement(ACCOUNT_BUTTON));
+//    }
+
+//    public void openContactsPage() {
+//        wait.until(ExpectedConditions.elementToBeClickable(CONTACTS_BUTTON));
+//        JavascriptExecutor executor = (JavascriptExecutor) driver;
+//        executor.executeScript("arguments[0].click();", driver.findElement(CONTACTS_BUTTON));
+//    }
```
Comment (vchesnakoutest) [inline src/test/java/page/HomePage.java]: is this lines are not necessary it's better to remove them

File: src/test/java/page/LoginPage.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage(WebDriver driver) {
+        super(driver);
+    }
+
+    BasePage waitForPageOpened() {
+        return null;
```
Comment (vchesnakoutest) [inline src/test/java/page/LoginPage.java]: why return null ? make void and add correct waiter

**Review by vchesnakoutest — COMMENTED** [review]
please make fixes and merge

File: src/test/java/page/AccountListPage.java (hunk 1 above)
Comment (YurySuvalau) [reply src/test/java/page/AccountListPage.java]: fix

File: src/test/java/page/AccountListPage.java (hunk 2 above)
Comment (YurySuvalau) [reply src/test/java/page/AccountListPage.java]: fix

File: src/test/java/page/HomePage.java (hunk 3 above)
Comment (YurySuvalau) [reply src/test/java/page/HomePage.java]: deleted

**Pull request merged**